src/ex3/PokemonClient_cli.js

```diff
@@ -0,0 +1,23 @@
+import PokemonClient from "../ex1/scripts/PokemonClient.js";
+import fetch from "node-fetch";
+
+export default class PokemonClient_CLI extends PokemonClient {
```

Comment:
why do you need this file and class extension ?
just use the PokemonClient.js you already created in previous exercises